mlir/include/mlir/InitMIOpenPasses.h

```diff
@@ -0,0 +1,43 @@
+//===- InitMIOpenPasses.h - MLIR MIOpen Passes Registration -----------*- C++
+//-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines a helper to trigger the registration of all our custom
+// dialects and passes to the system.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_INITMIOPENPASSES_H_
+#define MLIR_INITMIOPENPASSES_H_
+
+#include "mlir/Conversion/MIOpenPasses.h"
+#include "mlir/Dialect/MIGraphX/Passes.h"
```

Comment:
Sorry, this is dead code. migraphx dialect doesn't have any pass.
include/...../Dialect/MIGraphX/pass.h, pass.td should be deleted and related part in the cmakefile also needs to be removed.
Of course, no pass to register.

Reply:
Just found mlir/lib/Dialect/MIGraphX/PassDetail.h also needs to be deleted
and its cmakefile needs to remove the file and dependency to the MLIRMIGraphXPassIncGen

Reply:
@jungpark-mlir - I've gone and deleted all the stuff